spec/models/beer_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Beer, type: :model do
+
+  let(:beer) { build(:beer) }
+  subject { beer }
+
+  it { should be_valid }
+  it { should validate_presence_of :name }
+  it { should validate_uniqueness_of(:name).ignoring_case_sensitivity }
+
+  describe "Basic model properties" do
```

Comment:
Add a better indentation to this describe block please

Reply:
Ready @vicmaster, I already made the changes.

Reply:
use 2 black spaces instead of tabs